plugins/reference-assets/android/src/androidTest/java/com/intuit/player/android/reference/assets/input/InputTest.kt

```diff
@@ -53,12 +47,10 @@ class InputTest : AssetTest("input") {
         launchMock()
 
         val view = currentView.shouldBeView<ConstraintLayout>()
-        val inputLabelContainer = view.findViewById<FrameLayout>(R.id.input_label_container)
+        val inputNoteContainer = view.findViewById<LinearLayout>(R.id.input_note_container)
         val inputField = view.findViewById<FormattedEditText>(R.id.input_field)
 
-        inputLabelContainer[0].shouldBeView<TextView> {
```

Comment:
this is removing the check for the label container, but label isn't actually removed in the asset. Is it intentional to not check the label anymore?

Reply:
Issue resolved